2.0/s2i/bin/run

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+function export_memory_limit() {
+  if [ -f /sys/fs/cgroup/memory/memory.limit_in_bytes ]; then
+    export MEMORY_LIMIT_IN_BYTES=$(cat /sys/fs/cgroup/memory/memory.limit_in_bytes)
+    return 0
+  fi
+  return 1
+}
+
+function export_number_of_cores() {
+  if [ -f /sys/fs/cgroup/cpuset/cpuset.cpus ]; then
+    export NUMBER_OF_CORES=`awk ' \
+      BEGIN { i = 0; FS="-"; RS="[,\n]" } \
+      /^[0-9]+-[0-9]+$/ { i += $2-$1+1 }  \
+      /^[0-9]+$/        { ++i }           \
+      END { print i} \
+      ' /sys/fs/cgroup/cpuset/cpuset.cpus`
+    return 0
+  fi
+  return 1
+}
```

Comment:
@mfojtik yeah, me too :)

@mfojtik I don't want to use special values as errors as it's very messy. I can do with not echoing anything and the user needing to test for zero string. I guess that exporting it explicitly in the script and not inside of the function will make it easier for someone to find where the variable comes from..